security/MemberAuthenticator.php

```diff
@@ -51,7 +51,9 @@ protected static function authenticate_member($data, $form, &$success) {
 		if($asDefaultAdmin) {
 			// If logging is as default admin, ensure record is setup correctly
 			$member = Member::default_admin();
```

Comment:
I think the fix is in the wrong place. Member::default_admin() should NOT be returning null if a default admin is specified. that method is meant to be guaranteed to return an instance. Can you please investigate why this is not happening?